x-pack/plugins/synthetics/server/synthetics_service/formatters/icmp.ts

```diff
@@ -5,13 +5,12 @@
  * 2.0.
  */
 
-import { Formatter, commonFormatters, secondsToCronFormatter } from './common';
-import { ConfigKey, ICMPFields } from '../../../common/runtime_types/monitor_management';
+import { icmpFormatters as basicICMPFormatters } from '../../../common/formatters/icmp/formatters';
+import { Formatter, commonFormatters } from './common';
+import { ICMPFields } from '../../../common/runtime_types/monitor_management';
 
 export type ICMPFormatMap = Record<keyof ICMPFields, Formatter>;
-
 export const icmpFormatters: ICMPFormatMap = {
-  [ConfigKey.HOSTS]: null,
-  [ConfigKey.WAIT]: (fields) => secondsToCronFormatter(fields[ConfigKey.WAIT]),
+  ...basicICMPFormatters,
```

Comment:
I feel like basic should come after common, in terms of priority order for things that may need to be overwritten for a specific monitor type.